ledger/participant-integration-api/src/main/scala/platform/store/dao/JdbcLedgerDao.scala

```diff
@@ -720,7 +734,8 @@ private class JdbcLedgerDao(
       offsetStep: OffsetStep,
       packages: List[(Archive, PackageDetails)],
       optEntry: Option[PackageLedgerEntry],
-  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] =
+  )(implicit loggingContext: LoggingContext): Future[PersistenceResponse] = {
+    logger.info("Storing package entry")
```

Comment:
PackageLedgerEntry contains submissionId, please log it when provided

Reply:
Same comment as [above](https://github.com/digital-asset/daml/pull/8676#discussion_r571638038).